client/my-sites/plans/current-plan/index.jsx

```diff
@@ -163,9 +163,9 @@ class CurrentPlan extends Component {
 			translate,
 		} = this.props;
 
-		const currentPlanSlug = selectedSite.plan.product_slug;
```

Comment:
`ProductPurchaseFeaturesList` has `plan` as a required prop, and that's passed in here as `currentPlanSlug`. Maybe we should just render a placeholder on this page or something instead of `ProductPurchaseFeaturesList` if `currentPlanSlug` is not set?

Reply:
Apparently that prop isn't actually required (?) if the placeholder prop is set. I just updated `isLoading` to make sure that's correct at least.